packages/web-components/src/components/va-text-input/va-text-input.scss

```diff
@@ -24,15 +24,6 @@
   font-family: var(--font-source-sans);
 }
 
-:host(:focus) {
-  outline: none !important;
```

Comment:
The focus outline doesn't seem to change when I remove this style. It's still the desired yellow boarder. So in an effort keep unnecessary import keywords, I'm removing it. (I'm the one that added this to being with! 🫣)